src/main/java/Duke.java

```diff
@@ -1,10 +1,40 @@
+import command.CommandHandler;
+import constant.Constant;
+import parser.Parser;
+import storage.Storage;
+import java.util.Scanner;
+
+/**
+ * Main class for Mushroom Head.
+ */
 public class Duke {
+    private static final CommandHandler commandHandler = new CommandHandler();
+
+    /**
+     * This is the start of the main program.
```

Comment:
Try to follow the convention for header comments.
Something better would be `Starts the application ...`